samples/net/nats/sample.yaml

```diff
@@ -5,3 +5,12 @@ tests:
     harness: net
     depends_on: netif
     tags: net nats
+    platform_exclude: sam_e70_xplained
```

Comment:
dont like this, just settle on a default that works for everyone, this is just a sample, so please try to have 1 prj.conf.

Reply:
I would have done that if that would have been easy. The E70 ethernet driver checks that there is enough network buffers available and gives error if not. Unfortunately we have several prj.conf files in samples/tests that have less buffers configured.
This means that we either
* Remove the checks in E70 eth driver. I would like to avoid this as these checks make sense for this driver

or
* We increase the buf counts in various samples. This is not good either as then some of the smaller memory boards will start to fail

or
* Blacklist E70 in some of the failing tests. This is certainly an option but I preferred to just increase the buf counts for these couple of tests so that we get to compile them for E70.

If you have some other idea, please suggest.

Reply:
i would go with options 2:

> We increase the buf counts in various samples. This is not good either as then some of the smaller memory boards will start to fail